### DISCUSSION TITLE
Thank you very much for the fantastic experience in just 2 weeks as a GitHub member

### DISCUSSION THREAD

**Discussion by jakerevant** [community/community#57967] [New to GitHub]
I just signed up for GitHub around 2 weeks ago, set up my personal and my company GitHub accounts and a few projects I want to work on (I am working in IT for over 10 years now, am a certified multi-cloud solutions architect and DevOps engineer but did not really work with GitHub until now).

After some days my organizations were flagged, so I contacted support to ask why and to unflag those. The support was very friendly and helpful and all my organizations were unflagged again, as those have been wrongfully flagged by the automated systems. The understandable part of the answer also was, that I had created too many repos etc. for my projects too quickly after another. On the other hand, how should one do this then, I have 5 projects and shall wait 1 week or so until I start working on the next, because the automated system might think this could be something unusual?

Now a few days later, all organizations were flagged again, and I did not even access or work with them since I contacted support for the first time. So I replied to the email from support (as mentioned above) and told them that all my organizations have been flagged again. And now comes the absurd part, the answer was "my account was part of account farming efforts" and so this will be kept in place, and no further information whatsoever.

Sorry, but are you kidding me? First, I did not Google what account farming in regards to GitHub is, so please tell me.

Second, no, whatever it is, I simply signed up for GitHub to work on my new company's projects and to create a GitHub portfolio I can present to clients and partners. I even provided my full business information (also as billing information for the project I sponsor) to support in my first inquiry.

How can GitHub claim my account or organizations are part of "account farming efforts"? This is absolutely ridiculous. And it's especially worrisome that this is done to someone who is working on building a serious IT business and utilizing GitHub for this, as where does this make sense, I sign up to build a brand with my work and get treated this way? Isn't that a major part of what GitHub is also standing for, to showcase one's skills?

Thank you very much for the fantastic experience in just 2 weeks as a GitHub member.

**Comment by queenofcorgis** [comment]
Hi there - Thank you for sharing this feedback - it is our standard practice to follow the above protocol, but your account should now be released. We're glad to have you on GitHub and wish you a pleasant experience going forward!

**Reply by ettaboyle** [reply]
+1 to what @queenofcorgis said! We're happy to have you and appreciate the feedback. Please continue to use the Community Discussions to connect with our team and other users. Wishing you all the best!

**Reply by Bimlendu-Singh** [reply]
hello, i am a student doing graduation. I have a account in gihub (acc name- "Bimlendu-Singh"). it has been flagged out. I don't what i have done wrong because as per my knowledge i have repository only related to web dev and other coding languages. So what should i do know , please help. I have raised ticket to github support 3 days ago but i didn't get any response from them. Please help!
